From 7bec12b46cc40b2092b7859910ad22e03806a33f Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Sun, 15 Sep 2024 15:32:19 +0200
Subject: [PATCH 19/19] board: solidrun: hummingboard-t: enable support for
 legacy uboot image

Enable support for booting with legacy "boot.scr", e.g. to support
Debian out of the box.

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 configs/am64som_a53_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/am64som_a53_defconfig b/configs/am64som_a53_defconfig
index e0ccedd4db4..7f0e5cf3430 100644
--- a/configs/am64som_a53_defconfig
+++ b/configs/am64som_a53_defconfig
@@ -29,6 +29,7 @@ CONFIG_SPL_SPI=y
 # CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
 CONFIG_SPL_LOAD_FIT=y
 CONFIG_SPL_LOAD_FIT_ADDRESS=0x81000000
+CONFIG_LEGACY_IMAGE_FORMAT=y
 CONFIG_DISTRO_DEFAULTS=y
 CONFIG_BOOTCOMMAND="run envboot; run distro_bootcmd;"
 CONFIG_BOARD_LATE_INIT=y
-- 
2.43.0

